src/v2/Apps/SavedSearchAlerts/savedSearchAlertsRoutes.tsx

```diff
@@ -0,0 +1,50 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "v2/System/Router/Route"
+import { graphql } from "relay-runtime"
+
+const SavedSearchAlertsApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "savedSearchAlertsAppBundle" */ "./SavedSearchAlertsApp"
+    ),
+  {
+    resolveComponent: component => component.SavedSearchAlertsApp,
+  }
+)
+const OverviewRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "savedSearchAlertsAppBundle" */ "./SavedSearchAlertsOverviewRoute"
+    ),
+  {
+    resolveComponent: component =>
+      component.SavedSearchAlertsOverviewRouteFragmentContainer,
+  }
+)
+
+export const savedSearchAlertsRoutes: AppRouteConfig[] = [
+  {
+    path: "/user/alerts",
```

Comment:
Is there any chance we we can move all of this stuff to `Apps/Settings`? All user settings-related stuff should be located there for discoverability and co-location, even if it's not ultimately rendered under a `/settings/:foo` route. 

Thinking:
- `v2/Apps/Settings/SavedSearch`
- in `v2/Apps/Settings/settingsRoutes` you can then mount the `"/user/alerts", pointing at the components that now live in `v2/Apps/Settings/Routes/SavedSearch`

Reply:
done! Let me know what you think @damassi